README.md

```diff
@@ -172,6 +174,51 @@ jobs:
 We recommend you pin the version of Renovate to a full version or a full checksum, and use Renovate's regex manager to create PRs to update the pinned version.
 See `.github/workflows/build.yml` for an example of how to do this.
 
+### `docker-cmd-file`
+
+Specify a command to run when the image start. By default the image run
+`renovate`
+
+This option is useful to customize the image before running `renovate`
+
+For example you can create a simple script like this one (let's call it
+`renovate-entrypoint.sh`)
```

Comment:
```suggestion
Specify a command to run when the image start.
By default the image run
`renovate`.
This option is useful to customize the image before running `renovate`.
It must be an existing executable file on the local system.
It will be mounted to the docker container.

For example you can create a simple script like this one (let's call it
`renovate-entrypoint.sh`).

```